cmd/root_darwin.go

```diff
@@ -0,0 +1,3 @@
+package cmd
+
+const lineEnding string = "\n"
```

Comment:
these are all + more already captured in os

https://golang.org/pkg/os/#pkg-constants

os.PathSeparator

Reply:
I'm not looking for a path separator, I'm looking for the line endings specific to the OS, specifically when writing the gitignore file. This is a workaround I'd seen some people use since I couldn't find an os.LineEnding like I'm used to in C# or Java

Reply:
whew I definitely had path separators + using scanner instead of splitting on a separator, and those things got mashed together the mind yesh.

Reply:
follow up is why you need the different line endings? If we're not using that as a const to split on (since we should scan instead anyway) - the other option would be for writing line by line, in which i'm not sure we need to go that far - it seems like my understanding is just using \n is fine. It would also give the added benefit of us not having to deal with cross platform messiness with testing if we wanted to do just like byte-comparisons/hashes of expected vs actual output files since our output files would then only be `\n` separated.

Reply:
It was a scenario where I was going to write bytes to a file as "*" for the gitignore file and realized that "Holy crap, I need to have a different line ending for that component" since windows uses the \r\n sequence as opposed to just \n. Originally I was just going to leave it at "\n" but it felt prudent to deal with it there since I had thought of it as a problem. It leads to a crap ton of other questions to because technically on windows gitignore is handled by their compact bash shell that still technically parses the "\n" so it shouldn't matter in the end.